### DISCUSSION TITLE
Docker images on the docker hub are not updated

### DISCUSSION THREAD

**Discussion by paolomainardi** [squidfunk/mkdocs-material#7396] [General]
Is there a specific reason why the images on the Docker Hub are not updated as mentioned in the subject?
This is the latest tag pushed: https://hub.docker.com/layers/squidfunk/mkdocs-material/9.5.27/images/sha256-6e8bf9f36f9b5c9aa22def4fe91023f52f1c44d8c8fa49aa723d9746d89f2ec5?context=explore

**Comment by squidfunk** [comment]
Thanks for noting. There seems to be some problem with `msgpack`:
https://github.com/squidfunk/mkdocs-material/actions/runs/10054431955/job/27789011028#step:11:1246

I'll look into it.

**Reply by squidfunk** [reply]
I could reproduce the problem locally with:

```
docker buildx build --platform linux/arm/v7 -t squidfunk/mkdocs-material .
```

Adding `g++` as a dependency resolves the problem. Fixed in 627eb88b3. The Docker image will be updated on the next release.

**Reply by paolomainardi** [reply]
Thanks @squidfunk
